README.md

```diff
@@ -1,32 +1,51 @@
 Overview
 ========
 
-Token SDK for iOS. The SDK is built on top of protobuf gRPC API definition.
+Token SDK for iOS. Docs https://developer.token.io/sdk/
 
-Pick up new protos
-------------------
+The SDK is built on top of protobuf gRPC API definition,
+https://developer.token.io/sdk/pbdoc/io_token_proto_gateway.html
+
+Client Usage
+------------
+
+To use the SDK version compatible with Token's Sandbox testing
+environment (typical usage), use the published pod:
```

Comment:
not sure if apple-people say "publish" or if that's just java/maven-people

Reply:
It's ok I guess.

I find an example for your reference.
"To integrate AFNetworking into your Xcode project using CocoaPods"